internal/cmd/label/label.go

```diff
@@ -0,0 +1,104 @@
+package label
+
+import (
+	"fmt"
+	"github.com/Peripli/service-manager-cli/internal/cmd"
+	"github.com/Peripli/service-manager-cli/internal/output"
+	resperror "github.com/Peripli/service-manager-cli/pkg/errors"
+	"github.com/Peripli/service-manager-cli/pkg/types"
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/web"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+// Cmd wraps smctl label command
+type Cmd struct {
+	*cmd.Context
+
+	resourcePath string
+	id           string
+	labelChanges types.LabelChanges
+}
+
+// NewLabelCmd returns new label command with context
+func NewLabelCmd(context *cmd.Context) *Cmd {
+	return &Cmd{Context: context, labelChanges: types.LabelChanges{}}
+}
+
+// Prepare returns cobra command
+func (c *Cmd) Prepare(prepare cmd.PrepareFunc) *cobra.Command {
+	result := &cobra.Command{
+		Use:   "label [resource] [id] [operation] [key=value1,value2,...]",
```

Comment:
Just an idea don't know if its worth it:

What if the value of a label has "," in it? We can make the value be passed by a flag which is an array, this will allow for multiple values. The flag must have at least one value.

Reply:
What is appropriate design for the whole command and flag name if we include it?

I think `-k` for the key and `-v` for the value, where `-k` is single value flag and `-v` is an array is good, but `-v` is already in use for --verbose.

Maybe pass only values with flag, something like `-l` or `-val`?
How does this looks like to you?
`smctl label [resource] [id] [operation] [key] -l value1 -l value2 -l value3`